src/ResourceManager/Websites/Commands.Websites/help/New-AzureRmWebAppAzureStoragePath.md

```diff
@@ -0,0 +1,162 @@
+---
+external help file: Microsoft.Azure.Commands.Websites.dll-Help.xml
+Module Name: AzureRM.Websites
+online version:
```

Comment:
@vinisoto the build is currently failing because there's no value for this field. From the `ValidateHelpIssues.csv` file in the build artifacts:

_No entry was found for the online version field in the header. The corresponding URL should be the following: https://docs.microsoft.com/en-us/powershell/module/azurerm.websites/new-azurermwebappazurestoragepath_

Reply:
Thanks @cormacpayne - that solved it